snmp/datadog_checks/snmp/config.py

```diff
@@ -197,6 +203,13 @@ def add_profile_tag(self, profile_name):
         # type: (str) -> None
         self.tags.append('snmp_profile:{}'.format(profile_name))
 
+    def add_device_tags(self, device):
+        # type: (dict) -> None
+        for device_tag in SUPPORTED_DEVICE_TAGS:
+            tag = device.get(device_tag)
+            if tag:
+                self.tags.append('snmp_device_{}:{}'.format(device_tag, tag))
```

Comment:
I know we already have `snmp_host` and `snmp_profile`, but I'm not sure there is value of having `snmp_` prefix for tags (except for `snmp_host` since it collide with existing `host` tag).